docs/source/whatsnew/1.1.1.txt

```diff
@@ -1,23 +1,49 @@
-Development
------------
+Release 1.1.1
+-------------
 
 :Release: 1.1.1
-:Date: TBD
-
-.. warning::
-   This release is still under active development.  All changes listed are
-   subject to change at any time.
+:Date: June 30, 2017
 
 
 Highlights
 ~~~~~~~~~~
 
-None
+Since the the release of Zipline ``1.1.0``, Quantopian has officially released support for Futures in Zipline, as well as Python 3.5 support.
```

Comment:
As a reader, I find "since 1.1.0" ambiguous: does it mean these are features that first appeared in 1.1.0, or features that first appeared after 1.1.0?